routes/v1.js

```diff
@@ -87,6 +88,7 @@ function addRoutes(app, peliasConfig) {
       postProc.renamePlacenames(),
       postProc.parseBoundingBox(),
       postProc.normalizeParentIds(),
+      postProc.assignLabels(require('pelias-labels')),
```

Comment:
I don't think the intent of breaking out the label generation code into a separate repo is really to allow someone using the Pelias API to easily drop in their own label generation code. Instead it's to keep a really clear line of separation between the label generating code and everything else.

With that in mind it's nice to keep all knowledge of how to set up anything that _does_ require the label code confined to those modules. Right now the routes file needs to know not just to set up the label middleware, but what module to set it up with. I'd argue the label middleware should know how to set itself up.